.gitignore

```diff
@@ -61,3 +61,4 @@ lib/qtscript-bytearray/*.cc
 .idea
 *.vscode
 compile_commands.json
+.DS_Store
```

Comment:
`.gitignore` is not for files specific to your setup. It is for files that everyone working on Mixxx should ignore. Files specific to your setup should go in `.git/info/exclude`.

Reply:
I would submit that any GitHub repo that is intended to be used cross-platform should include `.DS_Store` in its `.gitignore` because the MacOS Finder creates and updates them, and you never want to commit them.

Reply:
I think that would be better handled by Git or the user's Git configuration.

Reply:
My thinking was that omitting it at the project level means that every inexperienced Mac user who works on the project is in danger of inadvertently committing inappropriate files to the repository. Since Apple will not listen to us.

Reply:
But yes, it would be nice to see at the Git level as well.

Reply:
I think it would make sense for Git to ignore `.DS_Store` by default on macOS...

Reply:
I agree, as discussed. We should remove all non-project related exclusions from our .gitignore.

Reply:
See also: #2255